### ISSUE TITLE
Protect contributors' privacy

### ISSUE THREAD

**Issue by hardcorekancil** [Yorubaname/yorubaname-website#17]
We can have the dictionary show only the usernames when it displays the name of the submitter. Therefore, current submissions will have the name of the submitter as something like kolatubosun@ while the rest is hidden, to prevent being exposed to privacy litigations.

**Comment by dadepo** [comment]
Instead of showing the email, we should be showing the names instead

**Comment by hardcorekancil** [comment]
Yes, that makes sense. Users should also be informed that their name is going to be displayed on the entry page and ideally have the option to display only the first name.
